### PR TITLE
feat: Integrate PNC script into tagRelease.sh

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
- tagRelease.sh should be invoking updatePNCBuildConfigs.sh script, as one will be almost always used with each other.
- minor formatting fix for updatePNCBuildConfigs.sh

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-3879

### CODE REVIEW

File: product/tagRelease.sh [hunk 1]
```diff
@@ -197,6 +198,22 @@ pushBranchAndOrTagGH () {
...
+  if [[ ! -x ${SCRIPTPATH}/updatePNCBuildConfigs.sh ]]; then
+    curl -sSLO https://raw.githubusercontent.com/redhat-developer/devspaces/${MIDSTM_BRANCH}/product/updatePNCBuildConfigs.sh --output-dir /tmp
+    chmod +x /tmp/updatePNCBuildConfigs.sh
+    PNC_SCRIPT_LOCATION="/tmp/updatePNCBuildConfigs.sh"
+  else
+    PNC_SCRIPT_LOCATION="${SCRIPTPATH}/updatePNCBuildConfigs.sh"
+  fi
+  # same source and target branch indicates the usecase for next workflow
+  if [ "${TARGET_BRANCH}" == "${SOURCE_BRANCH}" ];then
+    ${PNC_SCRIPT_LOCATION} ${DS_VERSION} --next
```
Comment (nickboldt) [inline product/tagRelease.sh:211]: you might be missing a `-v` flag here. https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/product/updatePNCBuildConfigs.sh#L29

File: product/tagRelease.sh [hunk 2]
```diff
@@ -255,5 +272,10 @@ for s in $sampleprojects; do
 	pushBranchAndOrTagGH "$s" ${samplesRepo}
 done
 
+# update PNC configs, only if performing branching operation
```
Comment (nickboldt) [inline product/tagRelease.sh:275]:
```suggestion
# update PNC build-configs, only if performing branching operation (not when tagging)
```

File: product/tagRelease.sh [hunk 3]
```diff
@@ -197,6 +198,22 @@ pushBranchAndOrTagGH () {
...
 
+updatePNCBuildConfigs() {
+  if [[ ! -x ${SCRIPTPATH}/updatePNCBuildConfigs.sh ]]; then
+    curl -sSLO https://raw.githubusercontent.com/redhat-developer/devspaces/${MIDSTM_BRANCH}/product/updatePNCBuildConfigs.sh --output-dir /tmp
+    chmod +x /tmp/updatePNCBuildConfigs.sh
+    PNC_SCRIPT_LOCATION="/tmp/updatePNCBuildConfigs.sh"
+  else
+    PNC_SCRIPT_LOCATION="${SCRIPTPATH}/updatePNCBuildConfigs.sh"
+  fi
+  # same source and target branch indicates the usecase for next workflow
```
Comment (nickboldt) [inline product/tagRelease.sh:209]:
```suggestion
  # if source and target branch are the same, we're updating the 3.x / next branch; else we're updating the 3.yy / latest branch
```

File: product/tagRelease.sh [hunk 4]
```diff
@@ -10,6 +10,7 @@
 
 # script to tag the Che/devspaces repos for a given release
 
+SCRIPT=$(readlink -f "$0"); SCRIPTPATH=$(dirname "$SCRIPT")
```
Comment (nickboldt) [inline product/tagRelease.sh:13]: can you also bump the copyright to 2023?

Comment (mkuznyetsov) [reply product/tagRelease.sh:14]: :+1: also updated usage text as well

**Pull request merged**